### PR TITLE
feat(docker): multiplatform buildx containers

### PR DESCRIPTION
**Pull request by JamesWoolfenden**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,9 +3,15 @@ FROM alpine:3.17.1
 # checkov:skip=CKV_DOCKER_2: Not a service
 # checkov:skip=CKV2_DOCKER_2: Not a service
 
+ARG BUILDARCH
+ARG DOWNTARGET="linux_${BUILDARCH}"
+RUN echo "${DOWNTARGET}"
```
Comment (gruebel) [inline Dockerfile:8]: added, but not used. any reason?

Comment (JamesWoolfenden) [reply Dockerfile]: cache invalidation

File: Dockerfile [hunk 2]
```diff
@@ -3,9 +3,14 @@ FROM alpine:3.17.1
 # checkov:skip=CKV_DOCKER_2: Not a service
 # checkov:skip=CKV2_DOCKER_2: Not a service
 
+ARG BUILDARCH
```
Comment (gruebel) [inline Dockerfile:6]:
```suggestion
ARG BUILDARCH=amd64
```
let's add a default, then it won't break during the docker build in GitHub Actions workflow 😄

**Review by gruebel — APPROVED** [review]
🍻

**Review by YaaraVerner — APPROVED** [review]

**Pull request merged**